src/oc-id/config/routes.rb

```diff
@@ -16,6 +16,7 @@
 
     resource :profile, only: [:show, :update] do
       put  "password" => "profiles#change_password"
+      put  "email" => "profiles#change_email"
```

Comment:
This is the bit that we decided to change from a put to a get - just to make accessing a link from an email easier.